rackcity/views/user_views.py

```diff
@@ -99,7 +103,28 @@ def user_list(request):
             status=HTTPStatus.BAD_REQUEST,
         )
 
-    users = User.objects.all()
+    users_query = User.objects
+
+    try:
+        filter_args = get_filter_arguments(request.data)
+    except Exception as error:
+        return JsonResponse(
+            {"failure_message": "Filter error: " + str(error)},
+            status=HTTPStatus.BAD_REQUEST
+        )
+    for filter_arg in filter_args:
+        users_query = users_query.filter(**filter_arg)
+
+    try:
+        sort_args = get_sort_arguments(request.data)
+        if len(sort_args) == 0:
+            sort_args = ['username']
```

Comment:
add default sort if none is specified to fix the weird behavior where modifying the admin permission of a user yeets them to the last row